config/core/configmaps/autoscaler.yaml

```diff
@@ -182,3 +182,9 @@ data:
     # allow-zero-initial-scale controls whether either the cluster-wide initial-scale flag,
     # or the "autoscaling.knative.dev/initialScale" annotation, can be set to 0.
     allow-zero-initial-scale: "false"
+
+    # max-scale is the cluster-wide default value for the max
```

Comment:
total, total nit, but I think the line length here is quite a lot shorter than in the rest of this configmap example block